.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -29,11 +31,9 @@
 -->
 - [x] None
 
-## Checklist:
-<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
-<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-- [ ] My change requires a change to the [documentation](https://github.com/hamlet-io/docs).
-- [ ] I have updated the documentation accordingly.
-- [ ] I have added tests to cover my changes.
-- [ ] All new and existing tests passed.
-- [ ] None of the above.
+## Checklist
+<!--- Review the following checklist to ensure you're Pull Request is complete.               -->
```

Comment:
Are you expecting people to answer these in the template? if you are I'm not sure this is required.